repo/fsrepo/migrations/ipfsfetcher/ipfsfetcher.go

```diff
@@ -287,7 +287,9 @@ func readIpfsConfig(repoRoot *string) (bootstrap []string, peers []peer.AddrInfo
 		return
 	}
 
-	cfgPath, err := config.Filename(*repoRoot)
+	// FIXME(BLOCKING): Decide if we need to add the config file path as
+	//  an extra variable in IpfsFetcher (and extend its constructor).
+	cfgPath, err := config.Filename(*repoRoot, "")
```

Comment:
Yeah we need this for migrations to work correctly right? AFAIU, nobody consumes this directly except us in go-ipfs and ipfs-update so we should be ok to break it.